student-store-ui/src/components/ReceiptModal/ReceiptModal.jsx

```diff
@@ -0,0 +1,30 @@
+import * as React from "react";
+import ProductCard from "../ProductCard/ProductCard";
+import "./ReceiptModal.css";
+
+export default function ReceiptModal({ name, products, total, clearReceipt }) {
+  return (
+    <div class="receipt-modal">
+      <div id="receipt-modal-content" class="receipt-modal-content">
+        <h3>{`Thanks for shopping, ${name}!`}</h3>
+        <div class="receipt-modal-products">
+          {products.map((product) => (
+            <ProductCard
+              product={product}
+              productId={product.id}
+              quantity={product.quantity}
```

Comment:
You're passing in the product, so the `ProductCard` could get the ID and quantity off of the `product` instead of passing in sub-fields.